File: MATEMÁTICA/1198/main.js
let lines = require('fs').readFileSync('/dev/stdin', 'utf-8').trim().split('\n');
for (let line of lines) {
    let [hashmat, opponent] = line.split(' ').map(Number);
    console.log(hashmat > opponent ? hashmat - opponent : opponent - hashmat);
}

File: MATEMÁTICA/1198/main.dart
import 'dart:io';

void main() {
  String? line;
  while ((line = stdin.readLineSync()) != null) {
    var [hashmat, opponent] = line!.split(' ').map(int.parse).toList();
    print(hashmat > opponent ? hashmat - opponent : opponent - hashmat);
  }
}
